lib/containers/common.pm

```diff
@@ -143,7 +143,9 @@ sub install_buildah_when_needed {
 
 sub install_containerd_when_needed {
     my $registry = registry_url();
-    zypper_call('in containerd cni-plugins', timeout => 300);
+    my @packages = qw(containerd);
+    push(@packages, 'cni-plugins') if is_sle("<15-SP7");
+    zypper_call('in ' . join(" ", @packages), timeout => 300);
     zypper_call('in -t pattern apparmor') if is_sle('=15-SP3');
```

Comment:
Can be merged with the zypper call above, just use the `pattern:apparmor` syntax